devel/tortoisehg/Portfile

```diff
@@ -18,23 +18,24 @@ description         A set of graphical tools for Mercurial
 long_description    A set of graphical tools for the Mercurial distributed \
                     source control management system.
 
-checksums           rmd160  9df80404cefa5567264913ea93ad778a4d971c70 \
-                    sha256  7c6d58b34678f237b0683dcac38707c91e10b46a28f918888fe6bb2c005c762c \
-                    size    8211370
+checksums           rmd160  bb915e3c207f82e19749b680bb0740c3fa9e5ca9 \
+                    sha256  5ae2cf9065bfbbd1491ff8bd4e895da92996da8622ef356a13b734e65199e62e \
+                    size    8219445
 
 python.default_version 27
 
 # Can use either mercurial or mercurial-devel.
 depends_lib         path:bin/hg:mercurial \
-                    port:py${python.version}-pyqt4 \
-                    port:py${python.version}-qscintilla-qt4 \
+                    port:py${python.version}-pyqt5 \
+                    port:py${python.version}-qscintilla-qt5 \
```

Comment:
This still needs to be fixed somehow (and was one of the reasons for having it WIP). We should only use Qt 5 on newer versions of macOS, perhaps 10.11 or 10.12. On older ones I would use Qt4 (or offer variants, but there's no need to make things too complicated).

Reply:
check